packages/docs/src/locales/en/en.json

```diff
@@ -144,7 +144,10 @@
         "indeterminateValue": "Overrides a state value that is not set.",
         "borderColor": "Color css style `border` (theme supported options or `HEX`).",
         "clearableIcon": "Sets the cleaning button icon.",
-        "tabindex": "Sets the custom `tabindex`."
+        "tabindex": "Sets the custom `tabindex`.",
```

Comment:
If it is not passed to the inner HTML element inside of component, I think it will be redundant, no?